capi/src/init-node/index.ts

```diff
@@ -60,9 +60,9 @@ const highlight = (code: string, language: string): string => {
     languageIsSet ? `-${language}` : ""
   }">${highlighted}</div>`;
 
-  return `<amplify-code-block language="${language}" line-count="${String(
```

Comment:
Although JS is famous for implicit type conversions, strings+numbers concatenation doesn't feel very deliberate. Is there a reason that you removed the explicit casting?

Reply:
Hmm, I'm not sure I have a great answer for you! I wanted to pass in a number, and not a String, to save us from the `parseInt()` business, but I see what you mean in that we're returning a string here.

Reply:
I put the explicit casting back, and uh, it still works. It's.. a string template, with a number in it, that gets coerced to a string... that... is a React tag.. that.. gets replace with... a component.. that... and then... uhhh.. takes a number... that was a string... uhh..

Help I'm on the tippy top of a teetering tower of complexity and I have no idea what I'm doing.

Let me know if this is better?